conf/app.routes

```diff
@@ -110,6 +110,7 @@ POST        /my-organisation/appoint-agent
 
 #MANAGE AGENT
 GET         /my-organisation/manage-agent                                                                                 @controllers.agent.ManageAgentController.manageAgent(agentCode: Option[Long] ?= None)
+GET         /my-organisation/show-agent/properties                                                                        @controllers.agent.ManageAgentController.showAgents()
```

Comment:
this URL isnt very descriptive in what the page is showing. /my-organisation/agents makes more sense.

Reply:
Changed route name